packages/fluentui/react-bindings/src/hooks/useStyles.ts

```diff
@@ -23,7 +23,7 @@ type UseStylesOptions<StyleProps extends PrimitiveProps> = {
    * A set props of that contain mapping for props that perform inline styles overrides, for example `styles` or
    * `variables`.
    */
-  mapPropsToInlineStyles?: () => InlineStyleProps<StyleProps>;
+  mapPropsToInlineStyles?: () => InlineStyleProps<any>;
```

Comment:
Is it possible to make this generic instead of using `any` ? 

```suggestion
  mapPropsToInlineStyles?: <ComponentProps,>() => InlineStyleProps<ComponentProps>;
```